h/sql_tasks/tasks/report/create_dev_users/01_create_schema.jinja2

```diff
@@ -0,0 +1 @@
+../create_from_scratch/00_schema/01_create.jinja2.sql
```

Comment:
You can't remove permissions on a schema if the schema doesn't exist. Part of the yak shaving required to create a user reliably.